nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -321,7 +321,7 @@ in
       enable = true;
       virtualHosts = mapAttrs (hostName: cfg:  mkMerge [ cfg.nginx {
         root = mkForce "${pkg hostName cfg}/share/dokuwiki";
-        extraConfig = "fastcgi_param HTTPS on;";
+        extraConfig = if cfg.nginx.addSSL || cfg.nginx.forceSSL || cfg.nginx.onlySSL || cfg.nginx.enableACME then "fastcgi_param HTTPS on;" else "";
```

Comment:
See `optionalString` instead of `if`.

Reply:
Thanks! We run an internal dns server, that is also not reachable from the internet.